src/core/client/admin/components/UserRole/UpdateUserRoleMutation.ts

```diff
@@ -44,12 +46,12 @@ const UpdateUserRoleMutation = createMutation(
             id: input.userID,
             role: input.role,
             moderationScopes: {
-              scoped: !!input.siteIDs?.length,
-              sites: input.siteIDs?.map((id) => ({ id })) || [],
+              scoped: !!input.scoped,
+              sites: [],
```

Comment:
as long as this doesn't cause a flicker between site moderater and moderator when changing a role, this should be fine.

Makes me wonder again how much we should be using optimistic responses instead of "updating" placeholders.

Reply:
I don't see any flicker on my machine, but we should make a note to check for this in the Coral party. I'm also open to "updating" placeholders.